marketplace/mainTemplate.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
I did a visual diff of this file vs [the sample file](https://github.com/Azure-Samples/kubernetes-offer-samples/blob/main/samples/k8s-offer-azure-vote/mainTemplate.json). It seems like the only difference is this file doesn't have `"app-title"` and `"app-value*`, right?

Reply:
Yes, that's correct.  The sample application ingested some custom values from the uidefinition (app-title, etc) but for this initial SpinKube offering, there aren't yet any values that need to flow through to the ARM template, etc.